### DISCUSSION TITLE
Use with existing github pages site?

### DISCUSSION THREAD

**Discussion by Sinclair-Speccy** [mkdocs/mkdocs#3402] [Q&A]
I was wondering if this can be used with a repo that is already using Github pages as I have a site that is already using Github pages, and don't want to make a whole new repo just for a manual I wish to create.

I've looked at the hosting guide for GitHub pages but this seems to only go over creating a new repo from the start. This is the repo that is already using GitHub pages: https://github.com/Sinclair-Speccy/The-Mainframe

I did try to do this the other day but when pushing the changes, it messed up the whole file structure of what was already there for my site. I am trying to get the stuff to stay in a specific folder

**Comment by oprypin** [comment]
Hi.
Sure you can do anything, because `mkdocs build` just produces a directory that you can put anywhere. Just don't use `gh-deploy`, it's an extremely limited tool that in the end just copies a directory.

If you're using GitHub Actions, here's an example of how I had 2 different directories on the same gh-pages site being produced by 2 separate systems (only one of them MkDocs) and separate workflows:

1. https://github.com/oprypin/crsfml/blob/2c660890c4efee633de71af2f422d75a2d257e20/.github/workflows/deploy-docs.yml#L26
2. https://github.com/oprypin/crsfml/blob/2c660890c4efee633de71af2f422d75a2d257e20/.github/workflows/deploy-book.yml#L34

**Reply by Sinclair-Speccy** [reply]
So I would just not need to use gh-deploy at all and it should stay in the same subdirectory for me where the files are? I tend to just dump the directory in where I want it to be
